bench/README.md

```diff
@@ -0,0 +1,56 @@
+# Running the performance benchmark
+
+## Required Files
+### CIFAR10 Python Dataset for SKLearn Model
+This benchmark serves an SKLearn model that depends on the CIFAR10 Python dataset for training. This dataset can be obtained from [https://www.cs.toronto.edu/~kriz/cifar.html](https://www.cs.toronto.edu/~kriz/cifar.html).

```

Comment:
This actually isn't true. The `cifar_utils` functions expect a CSV. You can use https://github.com/ucbrise/clipper/blob/develop/examples/tutorial/download_cifar.py to download the dataset and transform it into a CSV. If you look at the [tutorial](https://github.com/ucbrise/clipper/blob/develop/examples/tutorial/tutorial_part_one.ipynb), we first download the dataset and create the CSV, then load it to train the model.